hugo/config.toml

```diff
@@ -24,8 +24,8 @@ theme = "piercer"
 
 [params]
     description = "Wie siehts aus mit Klimaschutz in meiner Kommune?"
-    favicon = "/img/favicon.ico"
-    logo = "/img/klimawatch_gruen.png"
+    favicon = "img/favicon.ico"
+    logo = "img/klimawatch_gruen.png"
```

Comment:
habe gesehen, dass im preview diese beiden assets nun nicht mehr laden, glaube aber dass es so eigentlich korrekt ist.

Ich glaube wir haben in der gleichen Datei bei `baseurl` quatsch eingetragen. (Nämlich eher "' oder "https://klimawatch.codefor.de/")

Das sollten wir nach dem mergen mal direkt überprüfen

Reply:
Die `baseurl` Option, die hugo beim Aufrufen mitgegeben wird, überschreibt meines Wissens die Option in der `config.toml`. Entweder bei der `baseurl` oder bei den einzelnen Bildern müsste aber ein `/` auftauchen (vorne oder hinten, je nach dem). In der Preview fehlt jetzt der Slash, deswegen klappt es nicht.

Da man aber die `baseurl` im hugo-Aufruf für netlify-previews braucht (denn da ändert sie sich ja), würde ich die option aus der `config.toml` komplett rausschmeißen, um nicht zu verwirren. Kann man in der `netlify.toml` denn ein `/` anhängen? Dann könnte man den Rest der Pfade ohne `/` schreiben.

Reply:
https://github.com/codeformuenster/klimawatch/pull/29/commits/d100966cbba7993ed220702a52f2a25362ede406 scheints gefixt zu haben